include/sound/intel-nhlt.h

```diff
@@ -112,6 +112,11 @@ struct nhlt_vendor_dmic_array_config {
 	/* TODO add vendor mic config */
 } __packed;
 
+enum {
+	NHLT_CONFIG_TYPE_GENERIC,
+	NHLT_CONFIG_TYPE_MIC_ARRAY
```

Comment:
Assign values from spec?
```
eIntcConfigTypeGeneric  = 0,
eIntcConfigTypeMicArray = 1,
```

Reply:
I think that's implicit in C but I'll follow the suggestion.